GitCommands/ExternalLinks/ExternalLinksStorage.cs

```diff
@@ -73,7 +73,7 @@ public void Save(RepoDistSettings settings, IReadOnlyList<ExternalLinkDefinition
             }
         }
 
-        // TODO: refactor and outsource to the centralised SettingsSerialiser implementations.
+        // TODO: refactor and outsource to the centralised SettingsSerializer implementations.
```

Comment:
AE spelling suggested...

Reply:
I seem to remember checking that the new string matched an existing type name, but now I cannot find it. If this is supposed to be a type name, it should use `cref`.

Reply:
Depends on when the comment was made :) If the type doesn't currently exist - then perhaps should say "..centralised settings serializer implementations."